src/libraries/Common/src/Interop/Windows/SChannel/Interop.Alerts.cs

```diff
@@ -17,11 +17,8 @@ internal static partial class SChannel
         // to be passed into ApplyControlToken
         // through a PkgParams buffer.
 
-        public const int SCHANNEL_RENEGOTIATE = 0;   // renegotiate a connection
         public const int SCHANNEL_SHUTDOWN = 1;   // gracefully close down a connection
```

Comment:
Does it make sense to move these as well to Interop.Session.cs? so that they are all in the same place?